src/components/HdModal.vue

```diff
@@ -168,7 +168,8 @@ $mobile-browser-footer-buffer: 75px;
   padding: #{$sp-l + $sp-s} $sp-m $sp-m;
   background-color: $white;
   border-radius: 4px;
-  overflow: hidden;
+  max-height: 90%;
```

Comment:
I think that 100% should work because the `hd-modal` class already takes into account the padding on the bottom for Android.

Reply:
I will search about `env()` to fix this issue. Because 100% is not working properly on iOS Safari.